docs/articles/vs-test-adapter/Tips-And-Tricks.md

```diff
@@ -6,7 +6,7 @@ uid: tipsandtricks
 
 ## NUnit 3
 
-### VS Test .Runsettings configuration
+### DotNet test and VS Test .Runsettings configuration
```

Comment:
* Suggest chaining this to "dotnet run" (all lowercase) and surrounding that with backticks
* Suggest lowercase and backticks for .runsettings

Reply:
Not sure what you mean by your first bullet line there

Reply:
I added a comment with a suggestion to clarify 👍 sorry, couldn't do it easily on mobile when I was initially reviewing.